Sources/HTTPKit/Utilities/HTTPHeaderValue.swift

```diff
@@ -197,3 +197,26 @@ public struct HTTPHeaderValue {
         return HTTPHeaderValue(.init(value), parameters: parameters)
     }
 }
+
+extension HTTPHeaderValue: Codable {
```

Comment:
oh gosh, now GitHub thinks the rest of the file is a comment xD

Reply:
Yup, haha, they have a lot to learn about the new raw strings